.azure-pipelines/client.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
-
+          AZURE_TEST_MODE: "Playback"
```

Comment:
Does this belong in the `tests.yml` area instead of here?   You may want to include Wes or Daniel from engineering on the review and let them weigh in.

Reply:
I don't think so, at least not with the current default (`Live`). We want to switch to playback in PR tests only.

cc @weshaggard

Reply:
tests.yml currently is about live testing so i don't think we want to set it to playback there.